CMakeLists.txt

```diff
@@ -77,10 +77,17 @@ add_library(${PROJECT_NAME} INTERFACE)
 #  INTERFACE_COMPILE_FEATURES cxx_std_11
 #)
 
+option(USE_SYSTEM_WISE_TL_OPTIONAL "Use a system wise installation of tl-optional" OFF)
+
+if(USE_SYSTEM_WISE_TL_OPTIONAL)
+  find_package(tl-optional REQUIRED)
+endif()
+
 target_include_directories(${PROJECT_NAME} INTERFACE
-  "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include;${PROJECT_SOURCE_DIR}/external/lt>"
+  "$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include$<$<NOT:$<BOOL:${USE_SYSTEM_WISE_TL_OPTIONAL}>>:;${PROJECT_SOURCE_DIR}/external/tl>>"
```

Comment:
As per my previous comment, use `tl-optional_FOUND` instead of `USE_SYSTEM_WISE_TL_OPTIONAL`

Reply:
If `tl-optional_FOUND` is true and https://github.com/artivis/manif/pull/207#discussion_r649952287 is applied a systemwide installation of tl-optional will be used even if `USE_SYSTEM_WIDE_TL_OPTIONAL` is off. 

Is this one the desired behavior?

Reply:
If the system tl-optional is at least v.1, then I'd say yes this is the desirable behavior.